base/Player.ts

```diff
@@ -314,4 +314,14 @@ export abstract class Player {
             return false;
         }
     }
+
+    //Totally recovers character's HP, PP and status effects
+    total_recovery() {
+        this.current_hp = this.max_hp;
+        this.current_pp = this.max_pp;
+        for (let i = 0; i < 5; i++) {
```

Comment:
We don't need this for loop. All temporary statuses are removed at the battle end. Permanent statuses are removed by the healer. Then, since we're not totally recovering, maybe we could change this function name to `inn_recovery`.

Reply:
Ok, i'll rename the function. Should I mover It to InnMenu class or should It stay there?

Reply:
I think that would be better to move it to `InnMenu`.